config/toml.go

```diff
@@ -372,6 +361,18 @@ func ResetTestRootWithChainID(testName string, chainID string) *Config {
 	return config
 }
 
+func mustCreateTestRootDir(testName, chainID string) string {
+	homeDir := os.ExpandEnv("$HOME/.tendermint_test")
+	if err := os.MkdirAll(homeDir, 0700); err != nil {
+		cmn.PanicSanity(err.Error())
+	}
+	rootDir, err := ioutil.TempDir(homeDir, fmt.Sprintf("%s-%s_", chainID, testName))
+	if err != nil {
+		cmn.PanicSanity(err.Error())
```

Comment:
`cmn.PanicSanity` is deprecated. use `panic`

Reply:
will do, thanks